props/install/install.sh

```diff
@@ -7,14 +8,17 @@ mkdir ./berkeleyparser/
 wget https://github.com/emcnany/berkeleyinterface/archive/master.zip -O ./install/berekeleyinterface.zip
 wget https://github.com/astraw/svg_stack/archive/master.zip -O install/svg_stack.zip
 wget "http://downloads.sourceforge.net/project/jpype/JPype/0.5.4/JPype-0.5.4.2.zip?r=&ts=1459330071&use_mirror=nchc" -O install/JPype-0.5.4.2.zip
-wget http://berkeleyparser.googlecode.com/files/eng_sm6.gr -O ./berkeleyparser/eng_sm6.gr
-wget http://berkeleyparser.googlecode.com/files/BerkeleyParser-1.7.jar -O ./berkeleyparser/BerkeleyParser-1.7.jar
+wget "https://github.com/slavpetrov/berkeleyparser/raw/master/eng_sm6.gr" -O ./berkeleyparser/eng_sm6.gr
+wget "https://github.com/slavpetrov/berkeleyparser/raw/master/BerkeleyParser-1.7.jar" -O ./berkeleyparser/BerkeleyParser-1.7.jar
 wget https://repo1.maven.org/maven2/edu/stanford/nlp/stanford-corenlp/3.3.1/stanford-corenlp-3.3.1.jar -O ./dependency_tree/stanford-corenlp-3.3.1.jar
 cd install/
 unzip -o svg_stack.zip 
 unzip -o berekeleyinterface.zip
 unzip -o JPype-0.5.4.2.zip
 cd berkeleyinterface-master/
+# fix version compatibility issue between PropS, Berkeley Interface and setuptools
+# permanent fix should happen in Berkeley Interface repository, temporary fix until then:
+sed -i s/six==1.5.2/six==1.10.0/ requirements.txt
```

Comment:
Is this line needed if the the package version was also changed directly in the requirements file?